build/rpmfc.c

```diff
@@ -576,19 +569,16 @@ static void exclFini(struct exclreg_s *excl)
  * Run per-interpreter dependency helper.
  * @param fc		file classifier
  * @param ix		file index
- * @param nsdep		class name for interpreter (e.g. "perl")
- * @param depname	"provides" or "requires"
  * @param dsContext	RPMSENSE_FIND_PROVIDES or RPMSENSE_FIND_REQUIRES
  * @param tagN		RPMTAG_PROVIDENAME or RPMTAG_REQUIRENAME
  * @return		0 on success
  */
 static int rpmfcHelper(rpmfc fc, int ix, const struct exclreg_s *excl,
-		       const char *nsdep, const char *depname,
-		       rpmsenseFlags dsContext, rpmTagVal tagN)
+		       rpmsenseFlags dsContext, rpmTagVal tagN,
+		       const char *namespace, const char *cmd)
```

Comment:
The new params are not added to the doc string

Reply:
Urgh, yeah. Mind if I just nuke the doc string, it's just an internal helper after all...?

Reply:
Fine with me.

Reply:
Okay, axed them all in a separate commit.